Sources/SPMBuildCore/BuildParameters/BuildParameters.swift

```diff
@@ -225,9 +224,17 @@ public struct BuildParameters: Encodable {
 
     /// The path to the build directory (inside the data directory).
     public var buildPath: AbsolutePath {
-        if isXcodeBuildSystemEnabled {
-            return dataPath.appending(components: "Products", configuration.dirname.capitalized)
-        } else {
+        // TODO: query the build system for this.
```

Comment:
I filed https://github.com/swiftlang/swift-build/issues/286 to track the API needed for this.